packages/hardhat-plugin-ethers/src/ethers-ignition-helper.ts

```diff
@@ -145,6 +145,12 @@ export class EthersIgnitionHelper {
       defaultSender,
       strategy,
       strategyConfig: resolvedStrategyConfig,
+      maxFeePerGasLimit:
+        this._hre.config.networks[this._hre.network.name]?.ignition
+          .maxFeePerGasLimit,
```

Comment:
```suggestion
          ?.maxFeePerGasLimit,
```
Is this needed (and on the next one) for the case where ignition is defined but the `maxFeePerGasLimit` isn't?